great_expectations/expectations/core/schemas/UnexpectedRowsExpectation.json

```diff
@@ -68,6 +68,7 @@
         },
         "unexpected_rows_query": {
             "title": "Unexpected Rows Query",
+            "description": "A SQL or Spark-SQL query to be executed for validation.",
```

Comment:
We missed adding this field description to the schemas, but it's fine because it isn't used in GX Cloud for this Expectation.